files/zh-cn/web/javascript/reference/errors/missing_parenthesis_after_argument_list/index.md

```diff
@@ -28,7 +28,7 @@ console.log("PI: " Math.PI);
 // SyntaxError: missing ) after argument list
 ```
 
-你可以正确的调用 `log` 函数通过加上”+“操作符。
+你可以正确的调用 `log` 函数通过加上“+”操作符。
```

Comment:
Should this one be curly quotes or straight?

Reply:
curly quotes in Simplified Chinese.

Reply:
Chinese: --> “”(curly quotes)

Reply:
Very good, tnx 👍🏻